analytic_engine/src/instance/engine.rs

```diff
@@ -273,6 +277,7 @@ impl Instance {
         // Create space
         let space = Arc::new(Space::new(
             space_id,
+            schema_name,
             self.space_write_buffer_size,
             write_group,
             self.mem_usage_collector.clone(),
```

Comment:
I am not sure, if the `schema_name` should be placed in space.
As I see, schema shouldn't be equal to space.